libkbfs/folder_branch_ops.go

```diff
@@ -2231,6 +2248,12 @@ func (fbo *folderBranchOps) CreateFile(
 		entryType = File
 	}
 
+	if excl == WithEXCL {
+		if err = fbo.cr.Wait(ctx); err != nil {
```

Comment:
Do you think we should also retry, after ` cr.Wait()` call, if we get an `EXCLOnUnmergedError`?  Since we don't hold any locks here, it's very possible for another operation to slip in after this check and make us unmerged before taking the lock as part of `doMDWriteWithRetryUnlessCanceled`.  Not sure if it's worth doing or not.